### PR TITLE
Remove pydocstyle pins - there was an update to flake8-docstrings.

### PR DESCRIPTION
**Pull request by jmchilton**

### CODE REVIEW

**Comment by nsoranzo** [comment]
The new flake8-docstrings works, but it introduces a new check:
```
lib/galaxy/containers/__init__.py:63:1: D401 First line should be in imperative mood; try rephrasing
lib/galaxy/containers/__init__.py:68:1: D401 First line should be in imperative mood; try rephrasing
lib/galaxy/containers/docker.py:93:1: D401 First line should be in imperative mood; try rephrasing
```

**Comment by jmchilton** [comment]
Thanks @nsoranzo - taking another swipe at this.

**Comment by jmchilton** [comment]
Taking this out of WIP - looks like it fixes things now.

File: .ci/flake8_wrapper_docstrings.sh [hunk 1]
```diff
@@ -7,7 +7,7 @@ set -e
 # D3XX - Quoting issues.
 # D401 - First line should be in imperative mood 
 # D403 - First word of the first line should be properly capitalized
-args="--ignore=D --select=D100,D201,D202,D206,D207,D208,D209,D211,D3,D401,D403"
+args="--ignore=D --select=D100,D201,D202,D206,D207,D208,D209,D211,D3,D403"
```
Comment (nsoranzo) [inline .ci/flake8_wrapper_docstrings.sh:10]: By looking at the above comments, it seems that actually D401 was supposed to be enforced on the files included in `flake8_docstrings_include_list.txt` . Maybe remove `lib/galaxy/containers` from it instead?

Comment (jmchilton) [reply .ci/flake8_wrapper_docstrings.sh:10]: I considered that - I think lib/galaxy/containers previously conformed to that though right - this passed recently. So I guess the update changed what is considered "imperative mood" - and so we should probably re-address those comments and add this back in in a separate PR if it is important?

Comment (nsoranzo) [reply .ci/flake8_wrapper_docstrings.sh:10]: Right, this check was improved in https://github.com/PyCQA/pydocstyle/pull/235 .

**Pull request merged**